processing/src/main/java/org/apache/carbondata/processing/util/CarbonDataProcessorUtil.java

```diff
@@ -604,4 +606,33 @@ public static boolean isHeaderValid(String tableName, String header,
     }
     return dateformatsHashMap;
   }
+
```

Comment:
Module carbon-processing do not depends on spark or other computing engine however there are some class need multi-thread to load data which runs as computing engine's task which need get TaskContext using ThreadLocal tech.   

Yes, my first PR is merged from your PR333,but it's not merged to master yet.